### PR TITLE
Added cloud sql and new prod cluster

### PR DESCRIPTION
**Pull request by AlexGilleran**
### What this PR does
In _general_ the idea is to make it work with Google SQL and create a new prod cluster with higher availablility. What this involved:

* Separated database migrations from database images to facilitate use of managed SQL services - they now live in `magda-migrator-xx` directories and run as jobs on helm upgrade/install
* Added configuration for Google Cloud SQL
* Added a new helm config for the new cluster
* Bumped pods up to 2 replicas in this config where possible
* Normalised DB names - now everything is magda-xx-db
* Made docker build scripts automatically adjust `FROM` statements to add `localhost:5000/` and the correct version tag where necessary
* Deleted generated kubernetes files and gitignored them

### Checklist
- [x] Unit tests aren't applicable (delete one)
- [x] I've updated CHANGES.md with what I changed.
- [x] I've linked this PR to an issue in ZenHub (or there is no issue) and dragged it to the _Pull Request_ column

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 * Stopped the registry api from timing out and returning an error code when trimming by source tag / id - now returns 202 if it takes too long.
 * Added route for `/pages/*` requests so that `magda-web-server` won't response `Cannot GET /page/*`
 * Added format sleuther
-* Set `kubernetes-client` (required by magda-admin-api) version to `3.17.2` to sovle the travis build issue* Added ability to get records from the registry by the value of their aspects.* Added ability to get records from the registry by the value of their aspects.* Added access control layer to Authorization APIs: All `private` APIs (uri starts with /private/) can only be accessed by Admin users.
+* Set `kubernetes-client` (required by magda-admin-api) version to `3.17.2` to sovle the travis build issue* Added ability to get records from the registry by the value of their aspects.* Added ability to get records from the registry by the value of their aspects.\* Added access control layer to Authorization APIs: All `private` APIs (uri starts with /private/) can only be accessed by Admin users.
```
Comment (kring) [inline CHANGES.md]: Not new, but looks like there are three bullets on one line.  Also, the `\`, which is new, is extraneous.

File: deploy/helm/magda/charts/cloud-sql-proxy/templates/service.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+apiVersion: v1
+kind: Service
+metadata:
+  name: cloud-sql-proxy
+spec:
+  ports:
+  - name: postgres
+    port: 5432
+    targetPort: 5432
+{{- if .Values.global.exposeNodePorts }}
```
Comment (kring) [inline deploy/helm/magda/charts/cloud-sql-proxy/templates/service.yaml]: But this whole servie is only defined if `exposeNodePorts` is true. Is this check redundant, or is the check at the top an error?

File: magda-postgres/Dockerfile [hunk 3]
```diff
@@ -1,24 +1,21 @@
...
-RUN chown -R postgres /flyway
-COPY component/flyway-commandline-4.2.0-linux-x64.tar.gz /flyway
-COPY component/migrate-and-start.sh /usr/local/bin/
+COPY component/start.sh /usr/local/bin/
+# docker-entrypoint.sh (from the parent image) frustratingly only creates the database
+# if the "next" command it runs is "postgres".  But we want to do a migration after
+# the normal startup, so we can't simply launch postgres.  So here we monkey patch
+# the startup script to allow chaining to any command.  This is arguably better than
+# the alternative of forking the upstream docker-entrypoint.sh entirely.
+# COPY component/docker-entrypoint.sh.patch /usr/local/bin
```
Comment (kring) [inline magda-postgres/Dockerfile]: If none of this is needed anymore, probably better to just remove it.

File: magda-web-client/src/AppContainer.scss [hunk 4]
```diff
@@ -1,71 +1,81 @@
-@import './variables';
+@import "./variables";
```
Comment (kring) [inline magda-web-client/src/AppContainer.scss]: In the future, formatting edits like these should really be done in separate PRs from meaningful code changes, for the sake of review sanity.

Comment (AlexGilleran) [reply magda-web-client/src/AppContainer.scss]: [There was an attempt](https://github.com/TerriaJS/magda/commit/9bc2ec88c92039219eab2afa9db9c6a86a3d3fcd) :(

File: deploy/helm/magda/charts/cloud-sql-proxy/templates/service.yaml (hunk 2 above)
Comment (AlexGilleran) [reply deploy/helm/magda/charts/cloud-sql-proxy/templates/service.yaml]: Oops, yeah don't need that one.

File: magda-postgres/Dockerfile (hunk 3 above)
Comment (AlexGilleran) [reply magda-postgres/Dockerfile]: Done.

**Pull request merged**